app/src/main/res/layout/activity_create_config.xml

```diff
@@ -67,12 +75,16 @@
                     android:layout_height="wrap_content"
                     android:hint="@string/time_interval_hint"
                     android:imeOptions="actionDone"
+                    android:layout_marginTop="@dimen/create_config_margin1"
                     android:inputType="number"
                     android:textAlignment="center"
                     android:layout_gravity="center"/>
 
                 <android.support.v7.widget.AppCompatSpinner
                     android:id="@+id/interval_unit_spinner"
+                    android:layout_width="wrap_content"
+                    android:layout_gravity="center"
+                    android:layout_height="wrap_content" />
                     android:layout_width="@dimen/width_zero"
                     android:layout_weight="@dimen/weight_one"
                     android:layout_height="wrap_content"
```

Comment:
Please remove the lines off your PR if they are not being used in the code anymore